app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/optionsStepForm/optionsStepForm.jsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import CommentIcon from 'common/img/comment-inline.svg';
+import { DefectTypeItemML } from 'pages/inside/common/defectTypeItemML';
+import { useSelector } from 'react-redux';
+import { defectTypesSelector } from 'controllers/project';
+import Parser from 'html-react-parser';
+import { ScrollWrapper } from 'components/main/scrollWrapper';
+import { ExecutionInfo } from 'pages/inside/logsPage/defectEditor/executionInfo';
+import { useIntl } from 'react-intl';
+import { messages } from './../../messages';
+import styles from './optionsStepForm.scss';
+
+const cx = classNames.bind(styles);
+
+export const OptionsStepForm = ({ state }) => {
+  const { formatMessage } = useIntl();
+  const defectTypes = Object.values(useSelector(defectTypesSelector)).flat();
+  const defectType = defectTypes.filter((type) => type.locator === state.issue.issueType)[0];
+
+  const renderCommentBlock = () => {
+    return (
+      <div className={cx('comment-block')}>
```

Comment:
Please adjust the space between elements. The empty space should be removed (see the screenshot below).
![image](https://user-images.githubusercontent.com/31861805/113548192-c638bd00-95f7-11eb-82b6-20aab4f4615f.png)